### PR TITLE
LPS-173255 Don't set Index when parameters is empty

### PR DESCRIPTION
**Pull request by ces-totrinh**
Hi @Vy-CES,
Please help review this PR and leave your comments. Thanks.
Ticket: https://issues.liferay.com/browse/LPS-173255

### CODE REVIEW

File: modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/OpenAPIUtil.java [hunk 1]
```diff
@@ -143,7 +143,9 @@ private static String _getOperationScope(Operation operation) {
 			sb.append(",");
 		}
 
-		sb.setIndex(sb.index() - 1);
+		if (!parameters.isEmpty()) {
```
Comment (Vy-Bui17) [inline modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/OpenAPIUtil.java]: Hi @tototrinh , according to our discussion please consider:
1. moving the checking condition to the top of the function. If parameters are empty, return an empty string for example StringPool.BLANK 
2. please double-check if we still need to add the empty string to the array list at line 59. Looks like it does not matter if the scope is empty here: https://github.com/Vy-CES/liferay-portal/blob/master/modules/apps/batch-planner/batch-planner-rest-impl/src/main/java/com/liferay/batch/planner/rest/internal/resource/v1_0/SiteScopeResourceImpl.java#L75

**Review by Vy-CES — CHANGES_REQUESTED** [review]
Hi @tototrinh, thank you for sending the P/R.
Could you please take a look at the comments and let me know if you find them useful?

File: modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/OpenAPIUtil.java (hunk 1 above)
Comment (Vy-Bui17) [reply modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/OpenAPIUtil.java]: Do we need to write or update the integration test for this method in OpenAPIUtilTest?

Comment (ces-totrinh) [reply modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/OpenAPIUtil.java]: Thanks @Vy-CES for your feedback. I updated the code, please help take a look again. Thanks.

**Comment by Vy-Bui17** [comment]
Forward here: https://github.com/liferay-frontend/liferay-portal/pull/3026

**Pull request closed without merging**